lib/presentation/widget/myspace/my_space_viewmodel.dart

```diff
@@ -405,20 +411,56 @@ class MySpaceViewModel extends BaseViewModel {
     };
   }
 
+  void fetchData() {
```

Comment:
I don't think that this function is necessary !

Reply:
I renamed it to getSorter to make it easier to understand. This function fetches the stored sort object, retrieves a list of documents, and then sorts them.